swap/src/cli/command.rs

```diff
@@ -64,10 +64,7 @@ where
         Err(e) => anyhow::bail!(e),
     };
 
-    let debug = args.debug;
-    let json = args.json;
-    let is_testnet = args.testnet;
-    let data = args.data;
+    let (debug, json, is_testnet, data) = (args.debug, args.json, args.testnet, args.data);
```

Comment:
Not convinced this is really a style improvement. I think it is actually harder to read.

Reply:
You're right, but my main reason for doing that was I felt that it was just a waste to assign each of them individually